dist/index.cjs.cjs

```diff
@@ -345,6 +345,78 @@ const toCamelCase = str => {
   });
 };
 
+/**
+ * @param {RegExpMatchArray} match An inline tag regexp match.
+ * @returns {string}
+ */
+function determineTextStyle(match) {
+  const {
+    separator,
+    text
+  } = match.groups;
+  const [, textEnd] = match.indices.groups.text;
+  const [tagStart] = match.indices.groups.tag;
+  if (!text) {
+    return 'plain';
+  } else if (separator === '|') {
+    return 'pipe';
+  } else if (textEnd < tagStart) {
+    return 'prefix';
+  }
+  return 'space';
+}
+
+/**
+ * Extracts inline tags from a description.
+ * @param {string} description
+ * @returns {InlineTag[]} Array of inline tags from the description.
+ */
+function parseDescription(description) {
+  const result = [];
+
+  // This could have been expressed in a single pattern,
+  // but having two avoids a potentially exponential time regex.
+
+  // eslint-disable-next-line prefer-regex-literals -- Need 'd' (indices) flag
```

Comment:
Because I want the indices of every match I need to use the "d" flag, but that yields a parser error from eslint - so I had to use the workaround of passing the flags via a string to the `RegExp` constructor.

Reply:
Sure. As long as you're providing the rationale like this, I assume that's fine.